man/systemd.network.xml

```diff
@@ -1247,7 +1247,9 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
       <varlistentry>
         <term><varname>TypeOfService=</varname></term>
         <listitem>
-          <para>Takes a number between 0 and 255 that specifies the type of service to match.</para>
+          <para>Takes a number between 0 and 255 that specifies the Type Of Service (TOS) to match. The Type
+          of Service is also known as the Differentiated Services Code Point (DSCP). Only six bits are used;
+          in practice this number will be between 0 and 63.</para>
```

Comment:
The field is slightly complicated. And, it is hard to explain in short. Maybe, better to add links to RFCs or Wikipedia?

How about something like the following?
```
Takes an unsigned integer in the range 0…255.
This specifies the Type of Service (ToS) field of packets to match.
The field can be used to specify precedence (the first 3 bits) and ToS (the next 3 bits).
The field can be also used to specify Differentiated Services Code Point (DSCP) (the first 6bits) and
Explicit Congestion Notification (ECN) (the last 2 bits).
See ... for more details.
```

Reply:
Done.